### ISSUE TITLE
Feature: Allow setting of custom browser revisions

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#474]
There may be issues with the latest chrome issues. Allow users to set a custom browser revision.

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.36](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.35...1.9.36) (2020-05-22)

- build(deps): bump eventemitter2 from 6.4.0 to 6.4.1 [`#463`](https://github.com/open-wa/wa-automate-nodejs/pull/463)
- build(deps-dev): bump @types/node from 14.0.4 to 14.0.5 [`#473`](https://github.com/open-wa/wa-automate-nodejs/pull/473)
- Release 1.9.36 [`4e2016b`](https://github.com/open-wa/wa-automate-nodejs/commit/4e2016b2e3c652516772c0a48fe8d108fb91bfe5)
- feature: added `browserRevision` to ConfigObject #474 #462 [`6300e70`](https://github.com/open-wa/wa-automate-nodejs/commit/6300e70112bae49cab7b4816d3043196063540a6)
- fix: Now can properly utilise multiple spinners at once, added shouldEmit optional param [`60fcda0`](https://github.com/open-wa/wa-automate-nodejs/commit/60fcda0961c2ce171a0129deac333a1906bd9ade)
- cI: attempt to fix release it config [`1bc6837`](https://github.com/open-wa/wa-automate-nodejs/commit/1bc683770d401d2f08b902f262db3b50c120f7eb)

**Comment by smashah** [comment]
You can now set the browser revision in the config object. The puppeteer browser fetcher will then download (or load if it's already downloaded) the browser. This is how to do it:

```javascript
create({
...
browserRevision:"737027"
...
});
```

Please note THIS WILL OVERRIDE `useChrome` AND `executablePath`.

If you use a very old browser revision, it will cause a lot of issues WHICH THERE IS NO SUPPORT FOR.

Read more here: https://open-wa.github.io/wa-automate-nodejs/interfaces/configobject.html#browserrevision
